src/ucp/core/ucp_mm.c

```diff
@@ -631,6 +655,8 @@ ucp_memh_get_slow(ucp_context_h context, void *address, size_t length,
         goto err_free_memh;
     }
 
+    memh->reg_id = context->next_memh_reg_id++;
```

Comment:
we already do that in `ucp_memh_init_uct_reg`, do we need it here as well?

Reply:
`ucp_memh_init_uct_reg` is called only from `ucp_memh_get_slow`/`ucp_memh_alloc`